### PR TITLE
Adds Smart Mobility tags and updates a tag for Arterial Management

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/13268
- https://github.com/cityofaustin/atd-data-tech/issues/13398

## Testing
**URL to test:** Local

**Steps to test:**
1. 🔑 Start your local Moped 
2. 🕵️ Inspect the tag options on any project's summary tab. Observe the three tags of concern:
- `Smart Mobility - Expression of Interest`
- `Smart Mobility - Pilot`
- `Arterial Management`

You can also peep the data dictionary to accomplish the same task 😄 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/migrations/1692145812500_set_fk_public_moped_proj_tags_tag_id/up.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+alter table "public"."moped_proj_tags" drop constraint "moped_proj_tags_tag_id_fkey",
+  add constraint "moped_proj_tags_tag_id_fkey"
+  foreign key ("tag_id")
+  references "public"."moped_tags"
+  ("id") on update cascade on delete cascade;
```
Comment (johnclary) [inline moped-database/migrations/1692145812500_set_fk_public_moped_proj_tags_tag_id/up.sql:5]: this is consistent with our philosophy about cascades. if a dev is deleting from this table they must have a good reason to do so—so allow it.

Comment (johnclary) [reply moped-database/migrations/1692145812500_set_fk_public_moped_proj_tags_tag_id/up.sql:5]: ☝️ it's necessary for the other down migration to work.

**Review by chiaberry — APPROVED** [review]
i have peeped the data dictionary! 🐣

**Review by mddilley — APPROVED** [review]
All looks good here when tested locally - ship it! 🚀

**Comment by johnclary** [comment]
thanks y'all!

**Pull request merged**